src/createRollupConfig.ts

```diff
@@ -128,7 +128,7 @@ export function createRollupConfig(opts: TsdxOptions) {
       },
       typescript({
         typescript: require('typescript'),
-        cacheRoot: `./.rts2_cache_${opts.format}`,
+        cacheRoot: `./node_modules/.cache/rollup-plugin-typescript2_${opts.format}`,
```

Comment:
```suggestion
        cacheRoot: `./node_modules/.cache/tsdx_${opts.format}`,
```

Reply:
Sure, if you want to use `tsdx` instead this time around. Amended that to the commit but went a step further as `tsdx/${opts.format}/` so it's all under the `tsdx` cache directory.